lighthouse-core/audits/aria-allowed-attr.js

```diff
@@ -17,7 +17,12 @@
 
 'use strict';
 
+/**
+ * @fileoverview Ensures ARIA attributes are allowed for an element's role
```

Comment:
Can you end all these descriptions with a period? I hate to be that guy, but it like not having a close brace in code!